src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -858,7 +858,7 @@ public static SingleClusterDatabaseCommand ReadSingleCommand(ClusterOperationCon
         }
 
 
-        public static IEnumerable<SingleClusterDatabaseCommand> ReadCommandsBatch<TTransaction>(TransactionOperationContext<TTransaction> context, string database, long? fromCount, long take = 128)
+        public static IEnumerable<SingleClusterDatabaseCommand> ReadCommandsBatch<TTransaction>(TransactionOperationContext<TTransaction> context, string database, long? fromCount, long take = 128, long? lastCompletedClusterTransactionIndex = null)
```

Comment:
for code consistency, please keep the same order of arguments.

Reply:
Done